packages/tcllib/build.sh

```diff
@@ -0,0 +1,36 @@
+TERMUX_PKG_HOMEPAGE=https://core.tcl-lang.org/tcllib/
+TERMUX_PKG_DESCRIPTION="Tcl Standard Library"
+TERMUX_PKG_LICENSE="custom"
+TERMUX_PKG_LICENSE_FILE="license.terms"
+TERMUX_PKG_MAINTAINER="@termux"
+TERMUX_PKG_VERSION=1.21
+TERMUX_PKG_SRCURL=https://core.tcl-lang.org/tcllib/uv/tcllib-$TERMUX_PKG_VERSION.tar.xz
+TERMUX_PKG_SHA256=10c7749e30fdd6092251930e8a1aa289b193a3b7f1abf17fee1d4fa89814762f
+TERMUX_PKG_DEPENDS="tcl"
+TERMUX_PKG_BUILD_IN_SRC=true
+TERMUX_PKG_AUTO_UPDATE=true
+TERMUX_PKG_UPDATE_METHOD=repology
+TERMUX_PKG_HOSTBUILD=true
```

Comment:
Is host-build really necessary? It seems that `host_build` step is skipped by the custom function of `termux_step_handle_hostbuild`.

Reply:
Not required, as you said.

The entire installing process is architecture-independent since we dont build `critcl`, I forgot about this.  The new push fixed these.